src/main/java/com/bookappstore/controller/BookController.java

```diff
@@ -51,14 +55,24 @@ public List<BookDto> search(BookSearchParametersDto bookSearchParametersDto) {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
```

Comment:
Be consistent, why in some places you use "hasRole('ROLE_ADMIN')" and in other "hasRole('ADMIN')")
Also, think about which endpoint could use USER and ADMIN?

Reply:
Yes, really my bad